src/main/java/com/cloudbees/hudson/plugins/folder/computed/FolderComputation.java

```diff
@@ -113,8 +112,12 @@ public void run() {
             listener.getLogger().println("Aborted");
             _result = Result.ABORTED;
         } catch (Exception x) {
-            // TODO skip stack trace for AbortException
-            x.printStackTrace(listener.fatalError("Failed to recompute children of " + getDisplayName()));
+            String message = "Failed to recompute children of " + folder.getFullDisplayName();
+            if (x instanceof AbortException) {
+                listener.fatalError(message + ": " + x.getMessage());
```

Comment:
@jglick In this way your are **forcing** the message with *Failed to recompute children of*. I prefer something like that:

```
listener.fatalError("%n" + x.getMessage() + "%n%n");
```